spec/integrations/shop_features_spec.rb

```diff
@@ -7,14 +7,13 @@
 
 RSpec.describe 'Cart features' do
   let(:shop) { Shop.new }
-  let(:product) { Product.new('Szkolenie ruby', 100.20) }
+  let(:product) { Product.new(1, 'Szkolenie ruby', 100.20) }
+  let(:cart) { Cart.new.add_to_cart(product) }
 
-  describe 'user add a product' do
-    # subject { described_class.new }
-
-    it 'adds product to items list' do
-      @add = shop.add_product(product)
-      expect(@add).to eq([product])
+  describe '#show_products' do
```

Comment:
to chyba powinno być w `szop_spec` a tu ścieżka od utworzenia sklepu przez dodawanie i usuwanie produktów do sklepu i koszyka po wyczyszczenie koszyka